src/app/clusters/microwave-oven-control-server/microwave-oven-control-server.cpp

```diff
@@ -248,11 +248,16 @@ void Instance::HandleSetCookingParameters(HandlerContext & ctx, const Commands::
 
     if (startAfterSetting.HasValue())
     {
+        ConcreteCommandPath commandPath(mEndpointId, OperationalState::Id, OperationalState::Commands::Start::Id);
+
+#if CHIP_CONFIG_USE_EMBER_DATA_MODEL
```

Comment:
This seems like we should just have some sort of "server command exists" API on the IM engine, no?